omero/sysadmins/unix/install-web/install-fastcgi.txt

```diff
@@ -0,0 +1,120 @@
+OMERO.web FastCGI deployment
+============================
+
+.. _fastcgi_configuration:
+
+FastCGI Configuration (Unix/Linux)
+----------------------------------
+
+If you have installed Nginx or Apache OMERO can automatically generate a
+configuration file for your web server, see :ref:`apache22_configuration`,
+:ref:`apache24_configuration` or :ref:`nginx_configuration`.
+The location of the file will depend on your system, please refer to your
+web server's manual. See :ref:`customizing_your_omero_web_installation_unix`
+for additional customization options.
+
+
+.. _apache22_configuration:
+
+Apache 2.2 configuration (mod_fastcgi)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Install `mod\_fastcgi <http://www.fastcgi.com/drupal/>`_.
+To create a site configuration file for inclusion in the main Apache
+configuration redirect the output of the following command into a file:
+
+::
+
+    $ bin/omero web config apache
+
+.. literalinclude:: apache-omero.conf
+   :lines: 55-
+
+.. note::
+    The default configuration file installed with `mod_fastcgi` may be
+    incompatible with OMERO. In particular, the `FastCGIWrapper` option
+    conflicts with `FastCGIExternalServer` required by OMERO and must be
+    removed or commented out.
```

Comment:
Two note boxes are not very readable here - you could just have this first block of info as inline text under the included section, keeping just the note box below.